x-pack/test/accessibility/apps/search_profiler.ts

```diff
@@ -65,6 +70,11 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       await a11y.testAppSnapshot();
     });
 
+    it('close the flyout', async () => {
+      await testSubjects.click('euiFlyoutCloseButton');
```

Comment:
should this be part of the cleanup of the prior test, rather than its own separate a11y test?

Reply:
keeping it separate...same method followed in other tests as well.

Reply:
actually addressed review comments.

Reply:
I was mainly wondering if we're testing anything new here, i.e., after the flyout is closed, I imagine we're back in the same initial state that's tested in the `before()` block. But if this is the pattern we're following for a11y tests I'm OK with it it.